packages/sdk/src/index.ts

```diff
@@ -8,11 +8,11 @@ export { default as Multicall2 } from "./multicall2";
 
 // types
 import type { TypedEventFilterEthers as TypedEventFilter, TypedEventEthers as TypedEvent } from "@uma/contracts-node";
-import { Contract } from "ethers";
-import { Result } from "@ethersproject/abi";
```

Comment:
Does removing these impact the bundle sizes?

Reply:
no it doesnt, this was an artifact of me trying to remove the ethersproject dependency.  I will leave it in since i found that you can access these things from the `ethers` library itself.